webapp/apps/taxbrain/compute.py

```diff
@@ -96,14 +96,14 @@ def submit_calculation(self, mods, first_budget_year, url_template,
                            pack_up_user_mods=True,
                            additional_data={}):
         data = {}
-        if pack_up_user_mods:
-            user_mods = self.package_up_vars(mods, first_budget_year)
-            if not bool(user_mods):
-                return False
-            user_mods = {first_budget_year:user_mods}
-        else:
-            user_mods = mods
-
+        # if pack_up_user_mods:
```

Comment:
Feel free to go ahead and delete this code.

Reply:
Ok, will do. Thanks